favicon.md

```diff
@@ -0,0 +1,47 @@
+# Cómo generar un favicon
+
```

Comment:
Remember that when we often write guides it is best to assume that people are relatively new to the topic, unless we indicate it is for an advanced level. For the these guides, we should assume a newer audience. These guides will be their first point of reference. 

That written, it would add a lot of value to have a short description explaining `favicon` and its uses (for web and mobile browsers) and why we care about adding it to our pages.

Reply:
@Morban15 Excelente trabajando con el intro!